openedx/core/djangoapps/api_admin/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,99 @@
+"""
+Tests for the `api_admin` api module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import json
+
+from rest_framework.reverse import reverse
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from student.tests.factories import UserFactory
+from openedx.core.djangoapps.api_admin.tests import factories
+
+PASSWORD = UserFactory._DEFAULT_PASSWORD  # pylint: disable=protected-access
+
+
+class ApiAccessRequestViewTests(TestCase):
+    """
+    Tests for API access request api views.
+    """
+
+    def setUp(self):
+        """
+        Perform operations common to all test cases.
+        """
+        self.user = UserFactory.create()
+        self.client.login(username=self.user.username, password=PASSWORD)
```

Comment:
we can avoid the need to declare file variable `PASSWORD` and access the protected argument.
```python
self.password = 'test'
self.user = UserFactory.create(password=self.password)
self.client.login(username=self.user.username, password=self.password)
```